### ISSUE TITLE
Source image gets affected by multiply operation

### ISSUE THREAD

**Issue by fabiofidanza** [flrs/blend_modes#12]
I'm iterating over a set of images to blend them over a base image.
On the second iteration, the source image (which stays the same during the process) and the overlayed image (which gets reloaded at every iteration) becomes empty.
Is this by design?

```
i=0
for variant in variants: 
	warpedImage = np.zeros(baseLayer.shape, dtype=baseLayer.dtype)
	warpedImage[:,:,:] = 255

	variantImage = cv2.cvtColor(cv2.imread(variant).astype(np.float32),cv2.COLOR_RGB2RGBA);
	variantImage[:,:,3] = 255;

	cv2.warpPerspective(variantImage, transformMatrix, baseLayer.shape[:2],dst=warpedImage,borderMode=cv2.BORDER_TRANSPARENT)

	dstImage = bm.multiply(baseLayer,warpedImage,1)

	cv2.imwrite('images/output'+str(i)+'.png',dstImage)

	i += 1
```

**Comment by flrs** [comment]
I will check it out.

**Comment by flrs** [comment]
This is related to #7, specifically with the remark that the functions mutate their inputs. This is a bug.

**Comment by flrs** [comment]
Bug is fixed in release v2.1.0.
